script/run-local-cy-with-reports.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-console */
+/* run-local-cy-with-reports.js
+  * Script to run an app's Cypress specs locally with mochawesome reports
+  * Usage:
+  * yarn cy:run:localreports my-app-folder
+  */
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const cheerio = require('cheerio');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+const cyLocalReportsHtml = require('./get-cy-local-reports-index');
+
+// Get required spec-files parameter from command line arguments
+const appFolder = process.argv[2] || '';
+if (!appFolder) {
+  console.error(
+    chalk.red(
+      'ERROR: No app-folder parameter provided. You MUST provide an app-folder directly after the script name.',
+    ),
+  );
+  console.log(
+    chalk.yellow(
+      'Usage:\nyarn cy:run:localreports appeals/995\nyarn cy:run:localreports ask-a-question',
+    ),
+  );
+  process.exit(1);
+} else {
+  console.log(
+    chalk.yellow(`RUNNING CYPRESS WITH LOCAL REPORTS. appFolder: ${appFolder}`),
+  );
+}
+
+// Delete all output from previous test-runs
+try {
+  rimraf.sync('cypress/*');
```

Comment:
I think this should be fine, deleting all the files in these directories..., do you know what other scripts contribute files to these directories?

Reply:
No other _local_ script contributes to these git-ignored folders.  Cypress just creates this git-ignored folder in CI and outputs reports into it via a GitHub Action.  Since no GitHub Action even knows about our new script, we won't have any deletion-conflicts up there.